src/components/registration/Registration.tsx

```diff
@@ -35,51 +26,28 @@ type RegistrationProps = {
   onErrorFn: Function
 }
 
-const EMAIL_SIGN_IN_TRIGGER_ENDPOINT = '/v3/auth/email'
 const PHONE_SIGN_IN_TRIGGER_ENDPOINT = '/v3/auth/phone'
 
-const FLAGS = { greatBritain: 'GB', india: 'IN', southAfrica: 'ZA' }
-
-const signupIntro = {
-  PHONE: (
-    <Trans i18nKey="registration.intro1">
-      <h2>[translate]</h2>
-      <p>[translate]</p>
-      <p>[translate]</p>
-      <p>[translate]</p>
-    </Trans>
-  ),
-  EMAIL: (
-    <Trans i18nKey="registration.intro2">
-      <h2>[translate]</h2>
-      <p>[translate] </p>
-      <p> [translate]</p>
-    </Trans>
-  ),
-}
-
 export const Registration: React.FunctionComponent<RegistrationProps> = ({
   onSuccessFn,
   onErrorFn,
 }: RegistrationProps) => {
   const { t } = useTranslation()
   const stateSchema = {
-    email: { value: '', error: '' },
     phone: { value: '', error: '' },
-    registrationType: { value: 'PHONE', error: '' },
-    countryCode: { value: FLAGS.greatBritain, error: '' },
+    countryCode: {
+      value: window.localStorage.getItem('selected_country') || '',
+      error: '',
+    },
   }
 
   const validationStateSchema = {
-    //https://www.w3resource.com/javascript/form/email-validation.php
-    email: {},
     phone: {
       /*validator: {
```

Comment:
Can we remove commented code ? 
- if we think we can use this later, we can come to github history to get the code :)